### PR TITLE
fix xsd-fu/eclipse usage

### PR DESCRIPTION
**Pull request by joshmoore**
This should remove some of the project failures when
importing the project via m2eclipse.

### CODE REVIEW

File: components/ome-xml/pom.xml [hunk 1]
```diff
@@ -70,8 +70,8 @@
   <properties>
     <project.rootdir>${basedir}/../..</project.rootdir>
     <xsdfu.script>../xsd-fu/xsd-fu</xsdfu.script>
-    <xsdfu.modelpath>${basedir}/target/generated-sources</xsdfu.modelpath>
-    <xsdfu.metadatapath>${basedir}/target/generated-sources</xsdfu.metadatapath>
+    <xsdfu.modelpath>${basedir}/target/generated-sources/xsd-fu</xsdfu.modelpath>
+    <xsdfu.metadatapath>${basedir}/target/generated-sources/xsd-fu</xsdfu.metadatapath>
```
Comment (mtbc) [inline components/ome-xml/pom.xml]: FWIW adding this to the build path is helpful.

**Comment by joshmoore** [comment]
Re-adding to the build. (i.e. removing '--exclude') Happy for this to go in.

**Comment by mtbc** [comment]
This works well for me. I use this commit whenever I need to work with the ome-xml component.

**Comment by joshmoore** [comment]
Everyone's seems happy with it.

**Pull request merged**